laCour/slack-night-mode
# correct color of multi-party direct message text

Before: https://cl.ly/2J1s090X2T2y
After:  https://cl.ly/21293V1b0k1l

Test Plan:
- Send a direct message to a single person
- Send a direct message to a group of two or more people
- View your list of Direct Messages
- Verify that the names of people in multi-party messages is the same
  color as the text of single recipient messages
